data/LogicHelpers.json

```diff
@@ -17,22 +17,24 @@
 
     # Refill aliases. If we start considering Bow etc refills we can enable these aliases
     # without editing all logic files.
+    # The "limited" versions of certain items are less, or not at all, useful in logic.
```

Comment:
Most "limited" items are not in logic at all. Exceptions are limited bombchus (unlocks bombchu slot), limited skull mask (can be sold but not used in deku theater), and limited giant's knife (would be treated as permanently broken in logic). Bombchus are already listed, skull mask is handled in logic without need for an alias, and giant's knife has no logic.

Reply:
I may have misunderstood the function of the "limited" version items. But basically, what I was thinking, is that repeatable items placed in single-use chests have to become the limited version, at which point the logic ignores them. I guess that's okay for the majority cases, but I feel tunics should still be usable in logic and like-likes shouldn't eat them.

Reply:
I thought of adding an option to make tunics edible so that's why i added the non-repeatable versions, but they aren't used. I'm going to remove them.